### PR TITLE
Problems loading page

### PR DESCRIPTION
**Pull request by Kunal8208**
1. Added page showing Problems Bank for Admin

### CODE REVIEW

File: src/modules/admin/home/HomeComponent/index.jsx [hunk 1]
```diff
@@ -2,13 +2,13 @@ import PropTypes from 'prop-types';
 
 import AdminHeader from 'shared-components/AdminHeader';
 import SideNavContainer from 'modules/admin/sideNav/SideNavContainer';
+import { HOME_SCREENS } from 'modules/admin/home/HomeContainer/constants';
+import ProblemsListContainer from 'modules/admin/problemsList/ProblemsListContainer';
 import CreateDriveContainer from 'modules/admin/createDrive/CreateDriveCotainer';
 import CreateProblemContainer from 'modules/admin/createProblem/CreateProblemContainer';
 import AdminHomeConatiner from 'modules/admin/adminHome/AdminHomeContainer';
 import { Col, Container, Row } from 'core-components';
```
Comment (atharva-borekar) [inline src/modules/admin/home/HomeComponent/index.jsx:10]: change import order:
1. main libraries like react, prop-types
2. dependent libraries react-redux
3. components and containers, reducers
4. functions/methods like api calls
5. constants

File: src/modules/admin/problemsList/ProblemsListContainer/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import ProblemsListComponent from 'modules/admin/problemsList/ProblemsListComponent';
+import { useCallback, useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { ADMIN_ROUTES, ROUTES } from 'constants/routeConstants';
+import getProblems from 'modules/admin/createDrive/CreateDriveCotainer/getProblems';
+import { get } from 'redux/admin/apiHelper';
+import { SERVER_URL } from 'constants/appConstants';
+import { Alert } from 'core-components';
```
Comment (atharva-borekar) [inline src/modules/admin/problemsList/ProblemsListContainer/index.js:8]: import order

File: src/modules/admin/problemsList/ProblemsListComponent/index.jsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Table, Button, Spinner } from 'core-components';
+import './style.css';
+
+const ProblemsListComponent = (props) => {
+  const { renderTableData, problemIsLoading, handleAddProblemClick } = props;
+
```
Comment (Rutu2k) [inline src/modules/admin/problemsList/ProblemsListComponent/index.jsx:8]: (props) => {
  const { renderTableData, problemIsLoading, handleAddProblemClick } = props;
instead you can use this

({ renderTableData, problemIsLoading, handleAddProblemClick })

File: src/modules/admin/problemsList/ProblemsListContainer/index.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        id,
+        title,
+        description,
+      } = val;
+      return (
+        <tr key={id}>
+          <td>{id}</td>
+          <td>{title}</td>
+          <td>{description}</td>
+          <td>Null</td>
```
Comment (atharva-borekar) [inline src/modules/admin/problemsList/ProblemsListContainer/index.js:37]: Why does it have null written directly? Could you change it's value of what we get from api, or remove it if the component doesn't need it.

Comment (Kunal8208) [reply src/modules/admin/problemsList/ProblemsListContainer/index.js:37]: according to wireframe we have to show fileds (category, difficulty, marks) on front page table. But we are not receiving it from backend as of now. So I have made it Null for now.

**Pull request closed without merging**